### ISSUE TITLE
Sudden stuttering/low frame rate, closing Windows Audio Device Graph Isolation fixes this

### ISSUE THREAD

**Issue by CommieColin** [RPCS3/rpcs3#14648]
### Quick summary

RPCS3 will immediately start stuttering/performing slower than usual upon launching a game. Ending the Windows Audio Device Graph Isolation process in Task Manager immediately fixes the issue. Windows Audio Device Graph Isolation relaunches and is able to run with RPCS3 without causing issues. Something to do with CPU priority, perhaps? (Setting the rpcs3.exe priority to Real Time in Task Manager also seems to resolve this issue.) 

### Details

RPCS3 will immediately start stuttering/performing slower than usual upon launching a game. Audio is mostly not present and will pop in for one or two seconds before cutting out again. Ending the Windows Audio Device Graph Isolation process in Task Manager immediately fixes the issue. Windows Audio Device Graph Isolation relaunches and is able to run with RPCS3 without causing issues. Something to do with CPU priority, perhaps? (Setting the rpcs3.exe priority to Real Time in Task Manager also seems to resolve this issue.) 

Some additional info: 

Windows Audio Device Graph Isolation seems to be using ~1-2% of the CPU initially, after ending the process it relaunches itself and seems to be less CPU intensive. I have tried both Cubeb and XAudio2, the issue is the same on both. Enabling/disabling Audio Buffering and Time Stretching do not seem to have any effect. I do not have any microphones or audio input, all sound enhancements have been disabled on my external speakers. No audio issues with the game after ending the Windows Audio Device Graph Isolation process. Primary games I have experienced this on are Rock Band and Rock Band 2.

The attached logs are from launching the game, ending the Windows Audio Device Graph Isolation process in Task Manager and then exiting RPCS3 after the framerate has returned to normal. 

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/12623458/RPCS3.log)
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12623459/RPCS3.log.gz)

**Comment by Asinin3** [comment]
Logs dont capture performance, you should put a video.. but this seems weird anyways..

**Comment by Megamouse** [comment]
I think this is the same as #14517

**Comment by CommieColin** [comment]
> Logs dont capture performance, you should put a video.. but this seems weird anyways..

Good idea - video is attached. 

https://github.com/RPCS3/rpcs3/assets/94317308/e17b873c-c703-4148-a92d-4aa9fce84c9b

I know this PC is definitely a potato at this point, but as the video shows, Rock Band runs at a solid 60FPS normally. After reading through the comments on #14517, not sure if this is a regression of the RPCS3 build or a Windows issue. Hope the video helps.

**Comment by ReLyf47** [comment]
this is just like #14517 Jenci himself confirmed that this fixed the 1 thread dying
and it is a QT6 Issue on windows only

Very specific but
PC has to have 4 Thread (2C4T/4C4T)
any windows version Jenci has 10, and i tested mine on 11, happened in both case, QT5 doesnt have this issue

Could be a QT6+Windows compatibility thing, since on linux this doesnt happen

**Comment by CommieColin** [comment]
Interestingly enough, I noticed my audio device drivers were from Microsoft (guess I forgot to download the ones from my motherboard manufacturer's website when I did a clean install of W10 a while back) - I installed Realtek audio drivers and it seems to have resolved the issue for me; haven't had to kill Windows Audio Device Graph Isolation once since switching to the Realtek drivers. It's still running in the background, just without causing issues. 

Thanks to everyone for the insight and linking to #14517 - this is resolved for me at least. 

tl;dr - switched from Microsoft audio drivers to Realtek ones, seems to have resolved this issue.
